src/lib/storage/vector_compression/fixed_width_integer/fixed_width_integer_vector.hpp

```diff
@@ -57,8 +57,8 @@ class FixedWidthIntegerVector : public CompressedVector<FixedWidthIntegerVector<
     return _data.cend();
   }
 
-  std::unique_ptr<const BaseCompressedVector> on_copy_using_allocator(const PolymorphicAllocator<size_t>& alloc) const {
-    auto data_copy = pmr_vector<UnsignedIntType>{_data, alloc};
+  std::unique_ptr<const BaseCompressedVector> on_copy_using_memory_resource(MemoryResource* memory_resource) const {
```

Comment:
Why did you switch to a pointer instead of a reference?

Reply:
Reverted, using reference now